manifests/00-crd-extension-console-cli.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consoleclidownloads.console.openshift.io
+  annotations:
+    displayName: ConsoleCLIDownload
+    description: Extension for configuring openshift web console command line interface (CLI) downloads.
+spec:
+  group: console.openshift.io
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  scope: Cluster
+  names:
+    plural: consoleclidownloads
+    singular: consoleclidownload
+    kind: ConsoleCLIDownload
+    listKind: ConsoleCLIDownloadList
+  additionalPrinterColumns:
+  - name: Display name
+    type: string
+    JSONPath: .spec.displayName
+  - name: Age
+    type: string
+    JSONPath: .metadata.creationTimestamp
+  - name: Description
+    type: string
+    JSONPath: .spec.description
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console CLI download spec
+          required:
+          - displayName
+          - description
+          - link
+          properties:
+            displayName:
+              type: string
+              description: Display name of the CLI download
+            description:
+              type: string
+              description: Description of the CLI download (can include markdown)
+            link:
```

Comment:
Possibly needs to be `links [ ]`

Reply:
see https://github.com/openshift/console/pull/1441/files#r276345530

Reply:
Any reason to further consider this?  Would there be a `link` for the binary + a `link` for docs, for example?  @rhamilto @spadgett